h2o-parsers/h2o-orc-parser/build.gradle

```diff
@@ -3,28 +3,55 @@
 //
 description = "H2O Orc Parser"
 
+configurations{
+    // Configuration used to get all transitive dependencies for org.apache.hadoop:hadoop-common
+    // We do this to avoid manually excluding the libraries
+    hadoopCommonExclude
+    // The same for hive exec
+    hiveExecExclude
+}
+
 dependencies {
+  hadoopCommonExclude("org.apache.hadoop:hadoop-common:${defaultHadoopClientVersion}")
+  hiveExecExclude("org.apache.hive:hive-exec:1.2.2"){
+      // this dependency need to be excluded manually as Gradle can't find it in maven central
+      exclude group: 'org.pentaho', module: 'pentaho-aggdesigner-algorithm'
+  }
   compile project(":h2o-core")
   // Only PersistHDFS API
   compile(project(":h2o-persist-hdfs")) {
-    transitive = false
+      exclude group: 'ai.h2o', module: 'h2o-core'
+      exclude group: 'net.java.dev.jets3t', module: 'jets3t'
+      exclude group: 'org.apache.hadoop', module: 'hadoop-client'
+      exclude group: 'org.apache.hadoop', module: 'hadoop-aws'
   }
 
   // Note: What is connection between hive-exec version and hadoop-version and orc version?
-  // Note: In this case we are using hive version which is compatible with $orcDefaultHadoopClientVersion
+  // Note: In this case we are using hive version which is compatible with $defaultHadoopClientVersion
   // Note: for newest version it should be replaces by hive-orc
-  compile("org.apache.hive:hive-exec:$orcDefaultHiveExecVersion") {
-    transitive = false
+  compile("org.apache.hive:hive-exec:1.2.2") {
+      // we can't use transitive=false so we need to exclude the dependencies manually
+      configurations.hiveExecExclude.getResolvedConfiguration().getResolvedArtifacts().forEach {
```

Comment:
This is just automation so we don't have to list all transitive dependencies to exclude manually as there are lots of them for `hive` and `common` libs